rnc2rng/__main__.py

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env python
-from . import parser, serializer
+from __future__ import print_function
+
```

Comment:
I'd prefer to have to have the `print_function` import, then the other `from` import, then the `sys` import, on three consecutive lines, but I can fix this up myself if you prefer.